src/event/events.test.tsx

```diff
@@ -2,14 +2,23 @@ import React from "react";
 import { render, screen } from "@testing-library/react";
 import Event from "./event";
 
-test("renders the Event component", () => {
-  const title = "Event";
-  const description = "This is the Event Component";
+describe("Event", () => {
+  const props = {
+    title: "Test Title",
+    description: "Test Description",
+    start: new Date("2023-04-05T14:30:00Z"),
+    end: new Date("2023-04-05T16:30:00Z"),
+    url: "https://example.com",
+    email: "test@example.com",
+  };
 
-  render(<Event title={title} description={description} />);
+  it("renders the component with correct props", () => {
+    render(<Event {...props} />);
 
-  const elTitle = screen.getByText(title);
-  const elDesc = screen.getByText(description);
-  expect(elTitle).toBeInTheDocument();
-  expect(elDesc).toBeInTheDocument();
+    expect(screen.getByText(props.title)).toBeInTheDocument();
+    expect(screen.getByText(props.description)).toBeInTheDocument();
+    expect(screen.getByText("9:30 AM - 11:30 AM")).toBeInTheDocument();
```

Comment:
This test is currently non-deterministic and would fail if we ran it in a different timezone, how could we write this test differently, or update the function, to make this test pass no matter where it is run?